go.sum

```diff
@@ -1,7 +1,12 @@
 github.com/VividCortex/gohistogram v1.0.0 h1:6+hBz+qvs0JOrrNhhmR7lFxo5sINxBCGXrdtl/UvroE=
```

Comment:
Can we drop these files from the PR? I'd like to update dependencies in their own PR's.

Reply:
Yep